extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -161,7 +164,7 @@ export class Criteria {
    */
   static meetsSizingCriteria(element, renderWidth, renderHeight) {
     const {naturalWidth, naturalHeight} = getMaxNaturalDimensions(
-      dev().assertElement(element.querySelector('img'))
+      dev().assertElement(element.querySelector('img') || element)
```

Comment:
Increase test coverage to justify that `element` is an `img` and doesn't break otherwise.